src/CairoMakie.jl

```diff
@@ -522,25 +564,36 @@ function draw_atomic(scene::Scene, screen::CairoScreen, primitive::Text)
     broadcast_foreach(1:N, position, textsize, color, font, rotation) do i, p, ts, cc, f, r
         Cairo.save(ctx)
         char = txt[stridx]
+
         stridx = nextind(txt, stridx)
         rels = to_rel_scale(atlas, char, f, ts)
         pos = project_position(scene, p, Mat4f0(I))
         Cairo.move_to(ctx, pos[1], pos[2])
         Cairo.set_source_rgba(ctx, red(cc), green(cc), blue(cc), alpha(cc))
-        Cairo.select_font_face(
-            ctx, fontname(f),
-            Cairo.FONT_SLANT_NORMAL,
-            Cairo.FONT_WEIGHT_NORMAL
-        )
-        #set_ft_font(ctx, f)
-        ts = fontscale(atlas, scene, char, f, ts)
-        mat = scale_matrix(ts...)
+
+        cairoface = set_ft_font(ctx, f)
+
+        mat = if ts isa Number
+            scale_matrix(ts, ts)
+        else
+            scale_matrix(ts...)
+        end
         set_font_matrix(ctx, mat)
-        # set_font_size(ctx, 16)
+        
         # TODO this only works in 2d
         Cairo.rotate(ctx, -2acos(r[4]))
```

Comment:
```suggestion
        Cairo.rotate(ctx, -2acos(-r[4]))
```
We need to account for the fact that the traditional sign of the last term is inverted here.  With this, the `y` label seems to rotate properly...